sdmetrics/reports/single_table/_properties/coverage.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import pandas as pd
+import plotly.express as px
+
+from sdmetrics.reports.single_table._properties import BaseSingleTableProperty
+from sdmetrics.single_column import CategoryCoverage, RangeCoverage
+
+
+class Coverage(BaseSingleTableProperty):
+    """Coverage property class for single table.
+
+    This property assesses data coverage between the real and synthetic data.
+    A metric score is computed column-wise and the final score is the average over all columns.
+    The RangeCoverage metric is used for numerical and datetime columns while the CategoryCoverage
```

Comment:
minor: can you put RangeCoverage and CategoryCoverage in double backquotes like \``RangeCoverage\``